client/js/_author/components/common/upload_modal/meta_data.jsx

```diff
@@ -0,0 +1,84 @@
+import React      from 'react';
+import _          from 'lodash';
+
+export default class Metadata extends React.Component {
+  static propTypes = {
+    id: React.PropTypes.string.isRequired,
+    updateMetadata: React.PropTypes.func.isRequired,
+  };
+
+  constructor() {
+    super();
+    this.textArea = null;
+    this.state = {
+      baseScrollHeight: null,
+    };
+  }
+
+  areaResize() {
+    const minRows = 1;
+    this.textArea.rows = minRows;
+    const scrollHeight = this.textArea.scrollHeight;
+    if (!this.state.baseScrollHeight) { this.setState({ baseScrollHeight: scrollHeight }); }
+    const rows = Math.ceil((scrollHeight - (this.state.baseScrollHeight || scrollHeight)) / 17);
```

Comment:
Yup, because the guy who was doing this on the internet had it. And it worked, and I have no idea what it does, but it doesn't work w/o it